src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -1009,6 +1009,23 @@ class TestException : Exception { }
         End Function
 
         <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
```

Comment:
Add `<WorkItem(8623, "https://github.com/dotnet/roslyn/issues/8623")>` attribute so we know what issue this tests.